meta.yaml

```diff
@@ -0,0 +1,12 @@
+package:
+  name: tf-encrypted
+  version: "0.2.0"
```

Comment:
This should be added to the Makefile to ensure its updated with the `setup.py` and other files. In future this should probably be templated or something along those lines.

Reply:
how would you update this using the makefile?